include/cxxopts.hpp

```diff
@@ -582,168 +570,6 @@ struct ArguDesc {
   std::string value     = "";
 };
 
-#ifdef CXXOPTS_NO_REGEX
```

Comment:
this code was added not that long ago, I don't think we should remove it yet since it is providing support for fairly old compilers.
What was broken for you that would make you want to get rid of it?